pkg/test/ginkgo/duration_sorter.go

```diff
@@ -0,0 +1,64 @@
+package ginkgo
+
+import (
+	_ "embed"
+	"encoding/json"
+	"sort"
+
+	"github.com/sirupsen/logrus"
+)
+
+//go:embed testDurations.json
+var testDurationsData []byte
+
+// TestDurationData represents the duration information for a single test
+type TestDurationData struct {
```

Comment:
We have been talking about including more than just the duration data with the test names so we can consider things like pass rate over a period of time to consider dropping tests that never fail, etc.  That would be good to consider here as well if possible.